public/client/TracyProfiler.cpp

```diff
@@ -1462,6 +1463,22 @@ Profiler::Profiler()
         m_userPort = atoi( userPort );
     }
 
+    m_safeSendBufferSize = 65536;
+    m_safeSendBuffer = (char*)tracy_malloc( m_safeSendBufferSize );
+
+#ifndef _WIN32
+    pipe(m_pipe);
+#  if defined __APPLE__ || defined BSD
+    // FreeBSD/XNU don't have F_SETPIPE_SZ, so use the default 
+    m_pipeBufSize = 16384;
+#  else
+    m_pipeBufSize = (int)(ptrdiff_t)m_safeSendBufferSize;
```

Comment:
Why is double casting needed?

Reply:
I generally do this when both size and signedness are changing to (a) be explicit and (b) prevent pedantic warnings